cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -96,6 +83,7 @@ pub type LocationToAccountId = (
 	// Foreign chain account alias into local accounts according to a hash of their standard
 	// description.
 	HashedDescription<AccountId, DescribeFamily<DescribePalletTerminal>>,
+	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
```

Comment:
`DescribePalletTerminal` is included in `DescribeAllTerminal` so there's no need to have it explicitly put here.

```suggestion
	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
```

Reply:
changed.